docs/pages/api/signup.tsx

```diff
@@ -1,23 +1,31 @@
 import { NextApiRequest, NextApiResponse } from "next";
-import { subscribeToForm } from "../../lib/ConvertKitApi";
 
-const FORM_ID = process.env.CONVERTKIT_FORM_ID;
+const TRAY_URL = "https://39dca6c2-9ca4-41b4-82c9-e48202f221f8.trayapp.io";
```

Comment:
I can move this into an env var if that's preferred

Reply:
Yes please. Can you generate a new one and add it as env var since this URL is compromised now?